astropy/stats/tests/test_funcs.py

```diff
@@ -256,3 +256,35 @@ def test_gaussian_sigma_to_fwhm():
 def test_gaussian_sigma_to_fwhm_to_sigma():
     assert_allclose(funcs.gaussian_fwhm_to_sigma *
                     funcs.gaussian_sigma_to_fwhm, 1.0)
+
+def test_akaike_information_criterion():
+    #testing with arrays
+    aicc = funcs.akaike_information_criterion(np.array([-10,-5]), np.array([2,3]), 20, True)
+    assert_allclose(aicc, np.array([ 24.70588235, 17.5]))
```

Comment:
Not to be paranoid, but where to the numbers 24.70588 ... and 17.5 come from? Do you know that this example is true (e.g. from a trustworthy reference) or did you calculate them with your new code, in other words, were they calculated *assuming the code is right*? (In that case, the test still has some value because it prevents bugs that could will be introduced through code changes in the future, but obviously it would be best to test a "known good" example to make sure that the current implementation is right.)